apps/patient-documents/src/components/styles/DocumentsSection.module.scss

```
@@ -0,0 +1,84 @@
+.docRow {
+  padding-block: $spacing-05;
+}
+
+.typeCell {
+  @include type-style("body-compact-01");
+  color: $text-primary;
+}
+
+.note {
+  inline-size: 100%;
+  // Inset to match the row so the note box starts/ends with it.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)